swim/node.go

```diff
@@ -218,7 +218,7 @@ func (n *Node) Incarnation() int64 {
 
 func (n *Node) emit(event interface{}) {
 	for _, listener := range n.listeners {
-		go listener.HandleEvent(event)
```

Comment:
I'm blocking upon receiving certain events to synchronize the ping reqs in the tests in the right way.